src/ducks/classifications.js

```diff
@@ -121,10 +123,81 @@ const createClassification = () => {
   };
 };
 
+const saveAllQueuedClassifications = (dispatch) => {
+  const queue = JSON.parse(localStorage.getItem(FAILED_CLASSIFICATION_QUEUE_NAME));
+  if (queue && queue.length !== 0) {
+    console.log('Saving queued classifications:', queue.length);
+    queue.forEach((classificationData) => {
+      apiClient.type('classifications').create(classificationData).save()
+        .then((actualClassification) => {
+          console.log('Saved classification', actualClassification.id);
+
+          try {  //Fix: IE11 doesn't know what to do with Split.classificationCreated()
+            Split.classificationCreated(actualClassification);
+          } catch (err) { console.error('Split.classificationCreated() error: ', err); }
+
+          actualClassification.destroy();
+          const indexInQueue = queue.indexOf(classificationData);
+          queue.splice(indexInQueue, 1);
+          try {
+            localStorage.setItem(FAILED_CLASSIFICATION_QUEUE_NAME, JSON.stringify(queue));
+            console.info('Saved a queued classification, remaining:', queue.length);
+            console.info('ducks/classifications.js submitClassification() success');
+            dispatch({ type: SUBMIT_CLASSIFICATION_SUCCESS });
+          } catch (error) {
+            console.error('Failed to update classification queue:', error);
+          }
+          const { workflow, subjects } = actualClassification.links;
+          dispatch(addAlreadySeen(workflow, subjects));
+
+        })
+        .catch((error) => {
+          console.error('Failed to save a queued classification:', error);
+          console.error('ducks/classifications.js submitClassification() error: ', err);
```

Comment:
1. the `.catch()` argument is `error`, not `err` (should be named `err` for consistency)
2. This isn't `submitClassification()`, it's `saveAllQueuedClassifications()` (applies to other info logs in this function too)